### PR TITLE
Add Search parameter to Search action

### PR DESCRIPTION
**Pull request by Git-the-Sanz**
**What problem is this solving?**

Structured Data for the search isn't applying the appropriate [search parameter](https://help.vtex.com/tutorial/search-parameters--tutorials_567)

**How should this be manually tested?**

Access this [Workspace](https://searchparameter--hotpointuk.myvtex.com/)

Open developer tools > Elements > search for `potentialAction`
You'll now see the parameter applied to it.

### CODE REVIEW

File: react/SearchAction.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ function SearchAction({ searchTermPath }: Props) {
           url: baseUrl,
           potentialAction: {
             '@type': 'SearchAction',
-            target: `${baseUrl}${path}{search_term_string}`,
+            target: `${baseUrl}${path}{search_term_string}?map=ft`,
```
Comment (igorbrasileiro) [inline react/SearchAction.tsx:25]: @hiagolcm @thalytafabrine, maybe you have more context about the search... Can you take a look here?

**Review by thalytafabrine — APPROVED** [review]
lgtm, but it would be nice to have a review from someone with more context about this repository

**Review by vwraposo — APPROVED** [review]
LGTM

**Review by icazevedo — APPROVED** [review]
LGTM

**Pull request merged**